curvature/geomath.py

```diff
@@ -50,6 +50,8 @@ def distance_on_unit_sphere(lat1, long1, lat2, long2):
 
     cos = (math.sin(phi1)*math.sin(phi2)*math.cos(theta1 - theta2) +
```

Comment:
To clarify, this is where things went south, adding the two floats actually resulted in this number `1.0000000000000002` which is invalid to pass to `math.acos`